bosh-stemcell/lib/bosh/stemcell/stage_collection.rb

```diff
@@ -286,6 +288,28 @@ def photonos_os_stages
       ].flatten
     end
 
+    def opensuse_os_stages
+      [
+        :base_opensuse,
+        :base_runsvdir,
+        :base_file_permission,
+        :base_ssh,
+        :system_kernel_modules,
+        :system_ixgbevf,
+        bosh_steps,
+        :password_policies,
+        :restrict_su_command,
+        :tty_config,
+        :rsyslog_config,
+        :delay_monit_start,
+        :system_grub,
+        :cron_config,
+        :system_users,
+        :bosh_audit_opensuse,
```

Comment:
No, that just looks like an omission on our side. I pushed a commit which includes that stage.